README.adoc

```diff
@@ -44,6 +44,7 @@ that backend, and will avoid installing unneeded dependencies.
 from yubihsm import YubiHsm
 from yubihsm.defs import CAPABILITY, ALGORITHM
 from yubihsm.objects import AsymmetricKey
+from yubihsm.core import serialization    # importing this here allowed me to use it later
```

Comment:
Good catch! `serialization` comes from `cryptography` which is a dependency of `yubihsm`. Can you change the import to import directly from `cryptography` (i.e. `from cryptography.hazmat.primitives import serialization`)?